recipe/meta.yaml

```diff
@@ -51,8 +51,8 @@ outputs:
         - {{ pin_subpackage("cuda-crt-tools", exact=True) }}
         - {{ pin_subpackage("cuda-nvvm-tools", exact=True) }}
       run_constrained:
-        - gcc_impl_{{ target_platform }} >=6,<13  # [linux]
-        - arm-variant * {{ arm_variant_type }}    # [aarch64]
+        - gcc_impl_{{ target_platform }} >=6,<14.0a0  # [linux]
```

Comment:
Maybe we should put this in a Jinja variable up top near the version. Thinking this because

1. Place to set it
2. We can double check if it changed when making version updates

What do you think?

cc @adibbley (in case you have thoughts as well)

Reply:
This is definitely a good idea. I'll implement it.

Reply:
Done in 7b5962e.